src/templates/filtersDrawer.jsx

```diff
@@ -0,0 +1,674 @@
+                                <div
+                                    className="page_filters_drawer--clear_column"
+                                >
+                                    {/* eslint-disable max-len */}
+                                    {/* eslint-disable jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions, jsx-a11y/anchor-is-valid */}
+                                    {canClear ? (
+                                        <a
+                                            className={clearFiltersClasses}
+                                            onClick={this.onClearClick}
+                                        >
+                                            Clear Filters
+                                        </a>
+                                    ) : (
+                                        <Icon
+                                            color="static"
+                                            compact
+                                            size="medium"
+                                            type="filter"
+                                        />
+                                    )}
+                                    {/* eslint-enable jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions, jsx-a11y/anchor-is-valid */}
+                                    {/* eslint-enable max-len */}
+                                </div>
+
+                                <div
+                                    className="page_filters_drawer--apply_column"
+                                >
+                                    {isDirty && (
+                                    <Button
+                                        color="success"
+                                        icon
+                                        innerStyle={{
+                                            height: '32px',
+                                            width: '32px',
+                                        }}
+                                        onClick={this.onApplyClick}
+                                        style={{
+                                            height: '32px',
+                                            minHeight: '32px',
+                                            width: '32px',
+                                        }}
+                                    >
+                                        <Icon
+                                            compact
+                                            type="check"
+                                        />
+                                    </Button>
+                                    )}
+                                </div>
+                            </div>
+                        )}
+                    />
+
+                    <Drawer.Content>
+                        {_.map(rows, (row) => {
+                            rowKeyNum += 1;
```

Comment:
How is this any different from simply using the array index as a key? :-)
Not actionable; I'm sure it's just fine as-is.  I'm just curious about the thinking here.

Reply:
My thinking here is: don't ever use the index even if i know the list isn't going to change its sort order. Use an ID whenever possible and when not, use a global index.

Reply:
Okay, so, in your opinion, this is preferred in cases where there is no ID or other "natural" key to help reinforce that one should _never_ use the array key, period?   I'll buy that.

Reply:
Yep, that's my personal take. Doesn't have to be anyone else's though.

Reply:
I understand.  It does have the benefit of allowing all developers to look at the same code and _always_ know that "array indexes as keys are an anti-pattern".  That's an easier policy to grok than "sometimes using array indexes as keys are okay and other times it's bad; it depends!"

So ... I'm buying what you're selling.